src/main/java/quarkus/extensions/combinator/Configuration.java

```diff
@@ -33,7 +34,7 @@ public String get() {
             return Collections.emptyList();
         }
 
-        return Arrays.asList(get().split(","));
+        return Stream.of(get().split(",")).map(String::trim).collect(Collectors.toList());
```

Comment:
This change is to allow having spaces in the properties with comma-separated values. (Example: "A, B, C" now works, before, it does not).